### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by PricelessRabbit**
- fixed wrong mainflux url in doc
- added Agent start instruction in case of mainflux ui is used on the cloud

### CODE REVIEW

**Comment by nmarcetic** [comment]
@PricelessRabbit Please [sign-off commit](https://github.com/mainflux/agent/pull/45/checks?check_run_id=876607739), DCO is failing. Thanks!

**Comment by drasko** [comment]
@mteodor please review

File: README.md [hunk 1]
```diff
@@ -39,7 +39,15 @@ Start Agent with:
 ```bash
 MF_AGENT_BOOTSTRAP_ID=<bootstrap_id> \
 MF_AGENT_BOOTSTRAP_KEY=<bootstrap_key> \
-MF_AGENT_BOOTSTRAP_URL=https://mainflux.com/bs/things/bootstrap \
+MF_AGENT_BOOTSTRAP_URL=https://mainflux.com/things/bootstrap \
```
Comment (mteodor) [inline README.md:42]: this must be `http://mainflux.com:8204/things/bootstrap` if `bootstrap` is started additionally to the default docker composition ( and there is no mapping in nginx.conf for the bootstrap endpoint )

Comment (drasko) [reply README.md:42]: Wouldn't it be better to add mapping for this in NginX?

**Comment by manuio** [comment]
@PricelessRabbit DCO is failing

**Comment by drasko** [comment]
@PricelessRabbit please update this PR according to remarks

**Comment by drasko** [comment]
@PricelessRabbit please update the branch so we can merge.

@mteodor is this PR OK for merge? Please review.

**Comment by mteodor** [comment]
> @PricelessRabbit please update the branch so we can merge.
> 
> @mteodor is this PR OK for merge? Please review.

remarks not addressed

> @PricelessRabbit please update the branch so we can merge.
> 
> @mteodor is this PR OK for merge? Please review.

remarks not addressed

**Comment by manuio** [comment]
@PricelessRabbit please, update your branch

**Comment by PricelessRabbit** [comment]
OMG sorry for the delay. I updated the branch. It's ok or i have to do some changes to the doc?

**Comment by manuio** [comment]
> OMG sorry for the delay. I updated the branch. It's ok or i have to do some changes to the doc?

@PricelessRabbit looks like you have some unresolved remarks. @mteodor is this PR ok?

**Comment by PricelessRabbit** [comment]
> > OMG sorry for the delay. I updated the branch. It's ok or i have to do some changes to the doc?
> 
> @PricelessRabbit looks like you have some unresolved remarks. @mteodor is this PR ok?

Yeah at the time i changed the url in the doc to match the one it was working for me. I don't know if it is still valid or it is changes as the remarks  (the `http://mainflux.com:8204` was not working)

**Review by mteodor — APPROVED** [review]

**Review by drasko — APPROVED** [review]
LGTM

**Comment by drasko** [comment]
Merged!

Thanks @PricelessRabbit

**Pull request merged**